### DISCUSSION TITLE
Is it possible to use sidebar.yaml?

### DISCUSSION THREAD

**Discussion by dem1dov1van** [rohit-gohri/redocusaurus#359] [Q&A] [answered]
sidebar.yaml or any other way to pass a custom config for the sidebar? https://redocly.com/docs/developer-portal/configuration/sidebar-nav/

**Comment by rohit-gohri** [comment, marked as answer]
I think that is the docs of the SaaS offering of redoc called "Portals". 
All the options on this page - https://redocly.com/docs/redoc/config/ can be used with redocusaurus using redocly.yaml file in the root of the project
